understanding/20/audio-description-prerecorded.html

```diff
@@ -11,12 +11,12 @@ <h1>Understanding Audio Description (Prerecorded)</h1>
    <section id="brief">
 		<h2>In brief</h2>
 		<dl>
-		<dt>Objective</dt><dd>Videos can be played with audio descriptions</dd>
-		<dt>Author task</dt><dd>Provide a synchronized spoken description of the visual content in videos</dd>
-		
-		</dl>
-		
-		</section>
+         <dt>Goal</dt><dd>Videos can be played with audio descriptions.</dd>
+         <dt>What to do</dt><dd>Provide a synchronized spoken description of the visual content in videos.</dd>
+         <dt>Why it's important</dt><dd>People who are blind or have low vision can hear about visual content while playing videos.</dd> 
```

Comment:
@rainbreaw commented
> Not a concern, but a recommendation for consideration, in understanding/20/audio-description-or-media-alternative-prerecorded.html, understanding/20/audio-description-prerecorded.html and understanding/20/extended-audio-description-prerecorded.html:

> Consider extending why it's important to not just include people who are blind or with low vision, but also  "and individuals with a cognitive disability that makes it difficult to understand implied meaning such as facial expressions."

Reply:
Please see my [reply in audio-description-or-media-alternative](https://github.com/w3c/wcag/pull/3356/files#r1330597329). IMO, it's harder to work this into this 'why', and I'd like this one to stand. The Benefits section also specifically calls out "... those with cognitive limitations who have difficulty interpreting visually what is happening," so the consideration is present on the page.

Assuming you do pursue a PR to expand considerations in the Understanding document, I think it's also worth capturing that facial expressions may be reinforced sometimes with information provided in captions, which often provide context for the tone of what is said, as per this [guidance from 3playmedia](https://www.3playmedia.com/blog/dcmp-closed-captioning-standards/):
> Captions should preserve the speaker’s tone of voice when it is relevant for communication.

>For example, if a character’s emotion is not evident visually on screen (maybe they are offscreen, or their face is obscured), but their voice shows that they are angry, include [angrily] in their caption.

Reply:
I've thought about making this the following to better address your suggestion, but am a little concerned it is somewhat abstracting what is captured by audio descriptions.
```suggestion
         <dt>Why it's important</dt><dd>People who cannot see or understand the visual content can hear about it while playing videos.</dd> 
```

Reply:
@rainbreaw I think I'm good with the shorter rewording change I suggested, which can arguably capture all three groups, and I'm going to commit that. Thanks for all your input on this one!